packages/bpk-component-datatable/src/BpkDataTable.js

```diff
@@ -212,6 +236,9 @@ BpkDataTable.propTypes = {
   headerHeight: PropTypes.number,
   className: PropTypes.string,
   defaultColumnSortIndex: PropTypes.number,
+  sort: PropTypes.func,
+  sortBy: PropTypes.string,
+  sortDirection: PropTypes.string,
```

Comment:
```suggestion
  sortDirection: PropTypes.oneOf('ASC', 'DESC'),
```

Reply:
⬆️ this is because in the README we are choosing to be explict about the valid values